pycbc/inference/sampler/base_cube.py

```diff
@@ -75,8 +75,8 @@ def log_likelihood(self, cube):
         """
         params = dict(zip(self.model.sampling_params, cube))
         self.model.update(**params)
-        if self.model.logprior == -numpy.inf:
-            return -numpy.inf
+        #if self.model.logprior == -numpy.inf:
```

Comment:
@cdcapano This change is a question for you. I'm not really sure why this is here, and it isn't possible to make this work when you can't provide a logpdf function. For nested sampling, I would have though this would not be possible. Is this here to handle constraints or something like that?

Reply:
@cdcapano Can you clarify why this is done?

Reply:
@ahnitz Yes, this is to handle constraints. Since constraints can be arbitrary, you can't account for them when reshaping the hypercube. The only thing you can do is just treat points outside of the constraints as having 0 likelihood, which is what this line is enforcing. I don't follow, why is this an issue for this PR?

Reply:
@cdcapano As I said originally, there is no log prior function to provide. It will remain non implemented.

Reply:
@cdcapano To clarify more, we don't know the jacobian for a generic transformation which could be applied, so there is no logprior function at all for this class. The prior is only determined in the original space by sampling .

Reply:
@cdcapano Actually, I guess a correct thing to do would be to return a constant, right? It won't be used anyway and is technically correct (up to a constant which I won't calculate) for the sample parameters.

Reply:
Yeah, that looks fine.